src/main/resources/META-INF/plugin.xml

```diff
@@ -73,6 +73,12 @@
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
+            <action id="GitMachete.MergeParentIntoSelectedBranchAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.contextmenu.MergeParentIntoSelectedBranchAction"
+                    icon="MacheteIcons.MERGE_PARENT">
+                <override-text place="GitMacheteContextMenu"/>
```

Comment:
Also... for consistency with Rebase, maybe `Checkout and...` should be included in the descriptions?